irohad/main/irohad.cpp

```diff
@@ -159,6 +172,62 @@ static shared_model::crypto::Keypair getKeypairFromFile(
              });
 }
 
+boost::optional<std::string> getEnvVar(std::string const& key)
+{
```

Comment:
In other functions there is different style:
```suggestion
boost::optional<std::string> getEnvVar(std::string const& key) {
```

Reply:
We have clang-format for that. VSCode and CLion can format code using clang-format.

Reply:
@kuvaldini @baziorek Done: f327957c828321808ce57fc77cf78df82521c886